Assets/Scripts/Game/DaggerfallMissile.cs

```diff
@@ -183,6 +192,27 @@ private void Start()
                 }
             }
 
+            // Setup arrow
+            if (isArrow)
+            {
+                // Create and orient 3d arrow
+                goModel = GameObjectHelper.CreateDaggerfallMeshGameObject(99800, transform);
+
+                Vector3 adjust;
+                // Offset up so it comes from where it looks like it should from on the enemy sprites
+                if (caster != GameManager.Instance.PlayerEntityBehaviour)
+                    adjust = new Vector3(0, 0.6f, 0);
+                else // Offset forward to avoid collision with player and to the right to match bow animation
```

Comment:
Remember there's an option to switch weapons to left hand.. looks wierd with a bow but is possible.

Reply:
Thanks, fixing it now.

Reply:
This part is fine, as that option just draws the weapon to the left, but bows are still equipped in the right hand. But the arrow won't spawn in the right place, so I have to fix that.